packages/veraswap-sdk/script/utils/PoolUtils.sol

```diff
@@ -89,17 +82,17 @@ library PoolUtils {
             startingPrice,
             TickMath.getSqrtPriceAtTick(tickLower),
             TickMath.getSqrtPriceAtTick(tickUpper),
-            100 ether,
```

Comment:
on testnets it was nice for us to have more liquidity, can we make this a parameter on whether it's a native pool?
```bool isNative = tokenA == address(0) || tokenB == address(0)`